src/components/SqlModal.svelte

```diff
@@ -28,6 +25,75 @@
       }, 2000);
     }
   }
+
+  function getGlamSql() {
+    const osFilter =
+      $store.productDimensions.os === '*'
+        ? 'os IS NULL'
+        : `os="${$store.productDimensions.os}"`;
+    const buildIdFilter =
+      $store.productDimensions.aggregationLevel === 'build_id'
+        ? 'app_build_id IS NOT NULL'
+        : 'app_build_id IS NULL';
+    return _.template(desktopGlamSql)({
+      metric: $store.probe.name,
+      channel: $store.productDimensions.channel,
+      process: $store.productDimensions.process,
+      osFilter,
+      buildIdFilter,
+    });
+  }
+
+  function getTelemetrySql() {
+    // TODO: Use the time horizon to date window in SQL?
+    const { process } = $store.productDimensions;
+    let telemetryPath;
+    const table =
+      $store.productDimensions.channel === 'nightly'
+        ? 'main_nightly'
+        : 'main_1pct';
+    if ($store.probe.type === 'histogram') {
+      if (['main', 'parent'].includes(process)) {
+        telemetryPath = `payload.histograms.${$store.probe.name}`;
+      } else {
+        telemetryPath = `payload.processes.${process}.histograms.${$store.probe.name}`;
+      }
+    } else if ($store.probe.type === 'scalar') {
```

Comment:
I don't think we want this path? As we discussed the query won't work with scalars.

Reply:
Good catch, thanks.